pkg/proxy/proxy.go

```diff
@@ -21,7 +21,14 @@ import (
 // Public types
 //
 
-// Config provides the cfguration of a Proxy
+// EndpointRoute
+type EndpointRoute struct {
+	URL string
+
+	EventTypes []string
```

Comment:
Nit, but consider either (1) commenting these, or (2) tightening up the struct by killing newlines.